notebooks/vertex_genai/solutions/semantic_search_with_matching_engine_and_palm.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<p>It took me more than 30 mins to execute so maybe better to say that the execution more than 30 mins rather than up to 30 mins so no one is surprised when the execution too long. It took my cell 55 mins to execute.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I'll update now the time to reflect reality better. I'll try later to find a way to speed up training. Thanks!</p>